coverage.gradle

```diff
@@ -0,0 +1,90 @@
+apply plugin: 'jacoco'
+
+def isCore = (name == 'stream-chat-android-core')
+def testTask = isCore ? "test" : "testDebugUnitTest"
+def jacocoResults = "${buildDir}/reports/jacoco/report.xml"
+def ignoredModules = [
+        'stream-chat-android-ai-assistant',
+        'stream-chat-android-previewdata',
+        'stream-chat-android-docs',
+        'stream-chat-android-benchmark',
+        'stream-chat-android-compose-sample',
+        'stream-chat-android-e2e-test',
+        'stream-chat-android-test',
+        'stream-chat-android-ui-guides',
+        'stream-chat-android-ui-components-sample',
+        'stream-chat-android-pushprovider-firebase',
```

Comment:
This module doesn't exist anymore

Reply:
All 4 modules? So it's safe to delete these lines, isn't it?

Reply:
`stream-chat-android-pushprovider-firebase` is not a module within Stream Chat SDK anymore.
It was extracted to a different repository: https://github.com/getstream/stream-android-push

Reply:
Oh, I see, only `stream-chat-android-pushprovider-firebase` then. Will remove it shortly 👍

Reply:
Done ✅